api_v3/lib/types/filters/KalturaUserEntryFilter.php

```diff
@@ -58,7 +58,38 @@ protected function validateFilter()
 				$this->getFormattedPropertyNameWithClassName('userIdEqual') . '/' . $this->getFormattedPropertyNameWithClassName('userIdIn') . '/' .
 				$this->getFormattedPropertyNameWithClassName('entryIdEqual') . '/' . $this->getFormattedPropertyNameWithClassName('entryIdIn'));
 	}
-	
+
+	protected function getEntriesByEntitledKuserEdit()
+	{
+		$currentKsKuserId = kCurrentContext::getCurrentKsKuserId();
+
+		$allEntryIds = $this->entryIdEqual ? array($this->entryIdEqual) : explode(',', $this->entryIdIn);
+		$entries = entryPeer::retrieveByPKs($allEntryIds);
+		$entitledEntriesIdsForEdit = array();
+
+		/* @var $entry entry */
+		foreach ($entries as $entry)
+		{
+			if($entry->isEntitledKuserEdit($currentKsKuserId))
```

Comment:
Need to validate all the following - 
if ($entry->isEntitledKuserEdit($kuserId) || $entry->isEntitledKuserPublish($kuserId) || $entry->isEntitledKuserView($kuserId))
Each of this relations is explicitly set on the entry, and should be allowed.